### ISSUE TITLE
Some tests broken after switching to new dev computer

### ISSUE THREAD

**Issue by nlmarco** [cloudstore/cloudstore#46]
I switched to a brand new computer and a few tests were broken. I first suspected timing errors (due to the new machine being much faster), but most of the affected tests are absolutely not prone to such problems.

Two of the broken tests are:

LocalRepoManagerTest.checkParentLocalRevisionAfterChildAddition()
LocalRepoManagerTest.checkParentLocalRevisionAfterChildDeletion()

It seems the parent's directory is modified even though it should not.

**Comment by nlmarco** [comment]
I already fixed the bug before writing this issue :-D

The problem was that ext4 *may* store timestamps with a finer resolution than 1 ms. For whatever reason, the API of java.io.File and java.nio.* did not handle this the same way (at least that's the case with an OpenJDK 8). Therefore the code which previously used (at least sometimes) the old java.io.* API got things wrong and detected a modification (the timestamp was not the same after being written).

I now changed this to use the new java.nio.*-based API and it now works fine - both on our Jenkins and on my new dev machine.
